frontend/src/app/components/StreamlitDialog/__snapshots__/SettingsDialog.test.tsx.snap

```diff
@@ -47,14 +44,16 @@ exports[`SettingsDialog renders without crashing 1`] = `
           }
           value={0}
         />
-        <div>
+        <StyledButtonContainer
+          data-testid="edit-theme"
```

Comment:
Just added this `data-testid` to make it a little easier to get to the Edit active theme dialog